### PR TITLE
Show count of browse (Issue 106)

### PR DESCRIPTION
**Pull request by sk8higher**
Fixed buttons render (before that page crashed if you're not logged in)

### CODE REVIEW

File: app/views/vacancies/show.html.slim [hunk 1]
```diff
@@ -1,11 +1,14 @@
 .container.d-flex.flex-column.align-items-center
   h1 = @vacancy.title
+  - if user_signed_in? && current_user.id == @vacancy.user_id
```
Comment (XaoGao) [inline app/views/vacancies/show.html.slim:3]: @sk8higher I think will be better to create a new method, like 
```
user.owner?(@vacancy) 
```
write test

**Review by XaoGao — COMMENTED** [review]
@sk8higher pls fix

**Comment by sk8higher** [comment]
@XaoGao done

**Review by XaoGao — APPROVED** [review]
@sk8higher approve

**Pull request merged**